packaging/suse/velum.spec.in

```diff
@@ -56,7 +56,7 @@ BuildRequires:  %{rubygem bundler} >= 1.3.0
 
 # fixed gem dependent build requires
 BuildRequires: libxml2-devel libxslt-devel
-%if 0%{?suse_version} > 1500
+%if 0%{?suse_version} > 1500 || 0%{?sle_version} == 150000 
```

Comment:
i think to be fully correct we should chain the `is_opensuse` macro as well:

`%if 0%{?suse_version} > 1500 || ( 0%{?sle_version} == 150000 && 0%{?is_opensuse} )`

Reply:
thanks for the explanation :+1: